pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -370,14 +370,14 @@ func (p *staticPolicy) GetTopologyHints(s state.State, pod *v1.Pod, container *v
 	// Get a list of available CPUs.
 	available := p.assignableCPUs(s)
 
-	// Get a list of reusable CPUs (e.g. CPUs reused from initContainers). 
+	// Get a list of reusable CPUs (e.g. CPUs reused from initContainers).
 	// It should be an empty CPUSet for a newly created pod.
 	reusable := p.cpusToReuse[string(pod.UID)]
 
 	// Generate hints.
 	cpuHints := p.generateCPUTopologyHints(available, reusable, requested)
 	klog.Infof("[cpumanager] TopologyHints generated for pod '%v', container '%v': %v", format.Pod(pod), container.Name, cpuHints)
-	
+
```

Comment:
I see, it looks like you "fixed-up" all of these white-space changes at the end with a run of `update-all.sh`.
In the future, it would be nice to not include these at all along the way, so that they are not seen at all when reviewing commit-by-commit.

Reply:
It was unnecessarily introduced in refactoring. I squashed white-space changes to original commits.